### PR TITLE
Change CoreOS team permissions using an alias

### PR DESCRIPTION
**Pull request by jmarrero**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -62,3 +62,45 @@ aliases:
   - tsorya
   - osherdp
   - omertuc
+  coreos-approvers:
```
Comment (bgilbert) [inline OWNERS_ALIASES:65]: kelvinfan001, arithx, and zonggen are no longer on the team.  @ashcrow, should lorbuschris, sosiouxme, and jupierce be included in these lists?

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -62,3 +62,45 @@ aliases:
...
+  - cgwalters
+  - bgilbert
+  - dustymabe
+  - mike-nguyen
+  - jlebon
+  - ashcrow
+  - lucab
+  - sohankunkerkar
+  - travier
+  - kelvinfan001
```
Comment (miabbott) [inline OWNERS_ALIASES:76]: can drop Kelvin

**Comment by miabbott** [comment]
Couple of requested edits to the roster, but it looks like this will work

**Comment by hongkailiu** [comment]
The OWNERS file in `ci-operator` and in `core-services/prow/02_config` will be synced from the target repo by our automation unless it is ignored by

https://github.com/openshift/release/blob/64e72e6de535b715407ef2ff261806e0d7a1b9cd/ci-operator/jobs/infra-periodics.yaml#L980-L1010

```console
curl -s https://storage.googleapis.com/origin-ci-test/logs/periodic-prow-auto-owners/1448914270736617472/build-log.txt | grep "no OWNERS file" | rev | cut -d'=' -f 1 | rev | sort
3scale/3scale-operator
ComplianceAsCode/content
ViaQ/vector
che-incubator/che-test-harness
che-incubator/chectl
cloud-bulldozer/plow
containers/gvisor-tap-vsock
coreos/bootupd
coreos/coreos-assembler
coreos/rpm-ostree
devfile/registry
devfile/registry-operator
eclipse-che/che-dashboard
eclipse-che/che-operator
eclipse-che/che-plugin-registry
ibmz-eco/isv-operator
k8s-operatorhub/operators
kata-containers/kata-containers
kata-containers/runtime
monstorak/monstorak-operator
openshift-knative/kourier
openshift-kni/poison-pill
openshift/docker-distribution
openshift/dptp-workflow-test
openshift/etcd
openshift/hive-team
openshift/kubecsr
openshift/kubernetes-metrics-server
openshift/multiarch
openshift/openshift-docs
openshift/openshift-restclient-java
openshift/origin-metrics
openshift/origin-web-console-server
openshift/pagerduty-cluster-stats
openshift/project-request-limit
openshift/redhat-release-coreos
operator-framework/community-operators
ostreedev/ostree
red-hat-storage/rook
redhat-developer/git-service
redhat-openshift-ecosystem/community-operators-pipeline
redhat-openshift-ecosystem/community-operators-prod
redhat-openshift-ecosystem/cvp
redhat-openshift-ecosystem/playground
redhat-openshift-ecosystem/preflight
stackrox/rox-ci-image-openshift-ci-mirror
```

cross-check with the change files in this PR, the following files should not be modified.
- ci-operator/config/openshift/os/OWNERS
- ci-operator/config/openshift/rhcos-tools/OWNERS
- ci-operator/jobs/openshift/os/OWNERS
- ci-operator/jobs/openshift/rhcos-tools/OWNERS
- core-services/prow/02_config/openshift/os/OWNERS
- core-services/prow/02_config/openshift/rhcos-tools/OWNERS

**Review by hongkailiu — APPROVED** [review]

**Pull request merged**